### PR TITLE
shift circle angle a bit

### PR DESCRIPTION
**Pull request by MoKob**
# Issue

This PR is tuning the constants for turns in sharp curves just a tiny bit to circumvent a mix-up in fork directions at a specific intersection (see https://github.com/Project-OSRM/osrm-backend/issues/4331).

While this does not solve the underlying issue, it eliminates a wrong instruction at a prominent place.

## Tasklist
 - [ ] add regression / cucumber cases (see docs/testing.md)
        - I failed for more than an hour to get a test in cucumber that is exactly right. Given how specific and fragile the test would be, I currently cannot justify more time to create one.
 - [ ] review
 - [ ] adjust for comments

## Requirements / Relations
 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: src/extractor/guidance/coordinate_extractor.cpp [hunk 1]
```diff
@@ -122,15 +122,6 @@ util::Coordinate CoordinateExtractor::ExtractRepresentativeCoordinate(
     // coordinate set to add a small level of fault tolerance
     const constexpr double skipping_inaccuracies_distance = 2;
 
-    // fallback, mostly necessary for dead ends
```
Comment (MoKob) [inline src/extractor/guidance/coordinate_extractor.cpp]: this is a duplication branch with above that cannot be reached.

**Comment by daniel-j-h** [comment]
- I don't know how we come up with these constants
- They should probably be symbolized?

**Comment by MoKob** [comment]
@daniel-j-h by educated guessing...

**Comment by MoKob** [comment]
Capturing from chat:

This PR is all about choosing a small segment from the curve to represent the turn. The segment we had before moved one unit down the path.
A curved turn mostly feels straight, since the actual turn does not take a curve but the road after does.
Before we used the second 2-4 meters, now 4-6.
This is not a general fix but more a symptomatic change. The issue #4331 remains open.

**Pull request merged**